ipyrest/responseviews.py

```diff
@@ -190,6 +191,23 @@ def render(self, resp: requests.models.Response) -> Textarea:
         return ta
 
 
+class CSVResponseView(ResponseView):
+    """
+    A view that renders CSV data as an interactive table
+    """
+    name = 'CSV'
+    mimetype_pats = ['text/csv']
```

Comment:
Also, these MIME-type patterns must allow for additional information like in "text/csv; charset=utf-8".

Reply:
Right, so in this case `text/csv.*` should do the trick.